modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -20381,6 +20381,7 @@ x-is-a-reserved-word={0} is a reserved word and cannot be used.
 x-is-allowed-to-do-action-x-in-all-items-of-type-x-in-this-portal-instance={0} is allowed to do action "{1}" in all items of type {2} in this portal instance. Use the roles administration tool if you want to change this.
 x-is-allowed-to-do-action-x-in-all-items-of-type-x-in-x={0} is allowed to do action <em>{1}</em> in all items of type {2} in {3}. Use the roles administration tool if you want to change this.
 x-is-already-the-name-of-an-icon-pack="{0}" is already the name of an icon pack.
+x-is-being-used-by-a-root-object-and-cannot-be-deleted-to-delete-x-you-must-first-unbind-it={0} is being used by a Root Object and cannot be deleted. To delete {0}, you must first unbind it from the Root Object that is using it.
```

Comment:
@danvb112 To follow our current pattern, the language key should be written considering just the first sentence of its value.